mozilla-ai/lumigator
# Display Experiment results

## What's changing

- Table display for experiment results (different design from the previous tables)
- Click to copy experiment ID
- Highlight the selected item in both experiments' and datasets' tables
- Prevent user to create an experiment with datasets without ground_truth
- Allow streaming logs when an experiment is in progress

## How to test it

1. Upload a dataset without G_T
2. Try to trigger an experiment with that dataset( popup table menu, button inside dataset details, dropdown select in exepriment form)
3. Upload a dataset with G_T
4. Trigger an experiment
5. Observe status update both on experiments' table and experiment's details on the right side panel.
6. On experiment details copy the experiment id , use it in the openAPI for retrieving information(eg logs)
7. Open _View Results_ when experiment is finished

## Additional notes for reviewers
Try to navigate around while the experiment is running.

## I already...
Started working on notification service which will be available on the upcoming PR
